anthos-vmruntime/README.md

```diff
@@ -100,10 +100,10 @@ using `kubectl apply -f vmruntime.yaml`.
 
 Validate that the `VMRuntime` is enabled.
 ```sh
-kubectl describe vmruntime vmruntime | grep Ready
+kubectl wait --for=jsonpath='{.status.ready}'=true --timeout=300s vmruntime vmruntime
 
 # expected output
-  Ready:                       true
+vmruntime.vm.cluster.gke.io/vmruntime condition met
 ```
```

Comment:
Can we have the `expected output` in a separate code block? So clicking the copy button on the other one doesn't copy this.

Reply:
For sure